airbyte-integrations/bases/base-normalization/integration_tests/dbt_integration_test.py

```diff
@@ -160,8 +159,14 @@ def generate_profile_yaml_file(self, destination_type: DestinationType, test_roo
         profiles_config = config_generator.read_json_config(f"../secrets/{destination_type.value.lower()}.json")
         # Adapt credential file to look like destination config.json
         if destination_type.value == DestinationType.BIGQUERY.value:
-            profiles_config["credentials_json"] = json.dumps(profiles_config)
-            profiles_config["dataset_id"] = self.target_schema
+            credentials = profiles_config
+            profiles_config = {
+                "credentials_json": json.dumps(credentials),
+                "dataset_id": self.target_schema,
+                "project_id": credentials["project_id"],
+            }
+        elif destination_type.value == DestinationType.MYSQL.value:
+            profiles_config["database"] = self.target_schema
```

Comment:
These changes are necessary to fix tests due to: https://github.com/airbytehq/airbyte/pull/4699